qiita_pet/templates/study_ajax/prep_summary.html

```diff
@@ -107,8 +105,17 @@
     $('.prep-sample-delete:checkbox:checked').each(function(){
       sample_names.push($(this).val());
     });
-    sample_names = sample_names.join(',')
-    if(confirm("Are you sure you want to delete '" + sample_names + "?")) {
+    if (sample_names.length == 0) {
+      alert('No samples selected');
+      return false;
+    }
+    if (sample_names.length > 10) {
+      message = sample_names.length + ' samples';
```

Comment:
Can you declare the `message` variable i.e. `var message` somewhere at the beginning of the function? Otherwise this becomes global.